packages/core/src/lib/sign-in.ts

```diff
@@ -136,7 +137,11 @@ export const signInWithSocial = async (
     })
   );
 
-  const { id } = await findUserByIdentity(connectorId, userInfo.id);
+  const { id, identities } = await findUserByIdentity(connectorId, userInfo.id);
+  // Update social connector's user info
+  await updateUserById(id, {
+    identities: { ...identities, [connectorId]: { userId: userInfo.id, details: userInfo } },
```

Comment:
should we exclude `id` from `details`?

Reply:
`details` is flexable and is ok to save reductent infos.